web_monitoring/diffing_server.py

```diff
@@ -56,6 +57,7 @@
 
 
 client = tornado.httpclient.AsyncHTTPClient()
+debug_mode = (os.environ['DIFFING_SERVER_DEBUG'] == 'True')
```

Comment:
Three notes here:

- `os.environ['DIFFING_SERVER_DEBUG']` will throw a `KeyError` if `DIFFING_SERVER_DEBUG` is not set. I think this value should be optional, so you should do `os.environ.get('DIFFING_SERVER_DEBUG', 'False')`.

- With environment-provided booleans like this, I usually try and make them case-insensitive—otherwise, it’s really easy for someone to mis-configure the program and not know it. After all, how many times have you spent 15 minutes trying to figure out why something isn’t working only to discover it was a small typo? ;)

- Since this is a constant and we *generally* stick to [PEP 8](https://www.python.org/dev/peps/pep-0008/), please make it upper-case: `DEBUG_MODE`.

So:

```py
DEBUG_MODE = os.environ.get('DIFFING_SERVER_DEBUG', 'False').strip().lower() == 'true'
```

Reply:
Thank you for your help!